utils/tools.go

```diff
@@ -0,0 +1,35 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
not urgent for now, but we should think of a way to consume this package instead of copying it.

Reply:
Depends on future pipeline iterations, If we keep it, worth a shot.